bin/generate-shell-css.js

```diff
@@ -0,0 +1,25 @@
+var http = require( 'http' );
+var critical = require( 'critical' );
+
+// Load the HTML page from the dev server. This will fail if the dev server is
```

Comment:
This is fine for now, but the ideal solution would be to generate `shell.css` at build time, rendering `index.jade` and feeding it to `critical`. Have you considered that?